services/alerta/config.go

```diff
@@ -11,6 +11,8 @@ type Config struct {
 	Enabled bool `toml:"enabled" override:"enabled"`
 	// The Alerta URL.
 	URL string `toml:"url" override:"url"`
+	// Whether to skip the tls verification of the alerta host
+	TLSSkipVerify bool `toml:"tls_skip_verify" override:"tls_skip_verify"`
```

Comment:
Can you rename this option to `InsecureSkipVerify` and `insecure-skip-verify` so that it is consistent with other similar config options?